lib/autoyast.rb

```diff
@@ -23,6 +23,15 @@ def initialize(description)
       "repositories",
       "packages"
     )
+    if !description.users
+      Machinery::Ui.puts(
+        "\nWarning: Exporting a description without the scope 'users' as AutoYaST" \
+        " profile will result in a root account without a password which prevents" \
+        " logging in.\n" \
+        "So either inspect the scope 'users' before the export or" \
```

Comment:
Let my clarify: I didn't say that we should exit with an error. My comment was about rephrasing the warning, because this current text implies that system descriptions are always generated by inspect.